doc/users/next_whats_new/pass_pandasDataFrame_into_table.rst

```diff
@@ -0,0 +1,21 @@
+New Feature - passing Pandas.DataFrame into ax.table(...)
```

Comment:
```suggestion
``ax.table`` will accept a pandas dataframe
--------------------------------------------
```
This is going to be in the changelog under "What's new in Matplotlib 3.10.0" , so you don't need to say its a New Feature.